### ISSUE TITLE
Mocks

### ISSUE THREAD

**Issue by bketelsen** [bbqgophers/qpid#6]
Started a mocks directory.  Need to fill in mocks for testing without an actual grill.

**Comment by briandowns** [comment]
Any opinions on using Mockery? If not, as part of this task I can add any missing interfaces, run mockery, update tests using mocks, add a Makefile to do a `make mocks` or similar, etc.

**Comment by bketelsen** [comment]
Yeah I would like to take this opportunity to mock all the things and clean up
We’ll be working on this all week!
